torch/csrc/jit/ir/alias_analysis.cpp

```diff
@@ -173,16 +241,13 @@ bool AliasDb::hasWriters(const Value* v) const {
     return false;
   }
```

Comment:
Unrelated to this PR but: in `hasWriters` we're not testing the invariant that mutable values have an `Element *`